### ISSUE TITLE
readable.by not working on RHEL7

### ISSUE THREAD

**Issue by warroyo** [inspec/inspec#597]
I am attempting to use the `readable.by` and `readable.by_user` neither of which seems to be working. Here are the two problems I am seeing for each method.

readable.by: not sure if there is something else I need to set in order for this to work but `mask` is always nil

```
File /etc/at.allow should be executable by root
     Failure/Error: fail 'Invalid usergroup/owner provided' if mask.nil?

     RuntimeError:
       Invalid usergroup/owner provided
```

readable.by_user: This just hangs and does not return. I see the command running on the linux box but it is not returning.

**Comment by warroyo** [comment]
i needed to add the `--sudo` option

**Comment by chris-rock** [comment]
@warroyo Should we think about a better cli output that indicates a missing sudo option?

**Comment by warroyo** [comment]
Yes I think that would be great. The reason I did not find this sooner was due to mismatched documentation actually. The Chef Docs say there is a `--disable_sudo` option that is defaulted to false. and the documentation [here](https://github.com/chef/inspec/blob/master/docs/ctl_inspec.rst) says there is a `--sudo` option which is false. These two are conflicting so I thought I was using sudo the entire time because of the chef docs [here](https://docs.chef.io/ctl_inspec.html). However like you said, some better output form the cli would definitely help.
